.circleci/testnet-deploy-tag.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
```

Comment:
Same about nebula.

Reply:
This will be decided once you answer my comment regarding how to share the testnet configuration.
Perhaps we shall move the configuration of the CI testnet over to this repo and save us from the complexity of sharing a piece of a private repo.